### PR TITLE
Compress fonts

### PR DESCRIPTION
**Pull request by grod220**
Using woff2 format over ttf saves lots of kbs

### CODE REVIEW

File: packages/ui/styles/globals.css
Comment (grod220) [inline packages/ui/styles/globals.css:1]: Diffs here due to formatting

File: packages/ui/styles/globals.css [hunk 1]
```diff
@@ -3,149 +3,154 @@
...
+    @font-face {
+        font-family: 'Faktum';
+        font-weight: 600;
+        src: url('../fonts/Faktum-SemiBold.woff2') format('woff2');
+    }
+
+    @font-face {
+        font-family: 'Devanagari Sangam';
+        font-weight: 400;
+        src: url('../fonts/Devanagari-Sangam-Regular.woff2') format('woff2');
```
Comment (grod220) [inline packages/ui/styles/globals.css:124]: The key here is that ttf fonts are now using the woff2 formats

File: packages/ui/styles/globals.css
Comment (grod220) [inline packages/ui/styles/globals.css:1]: For some reason formatter did not like those kinds of comments to the side

**Review by VanishMax — APPROVED** [review]
wow, huge reduction of size!

**Pull request merged**